manifest/manifest.go

```diff
@@ -135,6 +135,10 @@ func GuessMIMEType(manifest []byte) string {
 		if ociMan.Config.MediaType == imgspecv1.MediaTypeImageConfig {
 			return imgspecv1.MediaTypeImageManifest
 		}
+		if ociMan.Config.MediaType == DockerV2Schema2ConfigMediaType {
+			return DockerV2Schema2MediaType
+		}
```

Comment:
Please document that this should not happen (`meta.MediaType` should have been set). 

(Use a `switch`?)